lib/Doctrine/DBAL/Migrations/Tools/Console/Command/StatusCommand.php

```diff
@@ -57,17 +57,20 @@ public function execute(InputInterface $input, OutputInterface $output)
     {
         $configuration = $this->getMigrationConfiguration($input, $output);
 
-        $currentVersion = $configuration->getCurrentVersion();
-        if ($currentVersion) {
-            $currentVersionFormatted = $configuration->formatVersion($currentVersion) . ' (<comment>'.$currentVersion.'</comment>)';
-        } else {
-            $currentVersionFormatted = 0;
-        }
-        $latestVersion = $configuration->getLatestVersion();
-        if ($latestVersion) {
-            $latestVersionFormatted = $configuration->formatVersion($latestVersion) . ' (<comment>'.$latestVersion.'</comment>)';
-        } else {
-            $latestVersionFormatted = 0;
+        $versionsFormatted = array();
```

Comment:
Done in https://github.com/doctrine/migrations/commit/155115f8b14e2f98e473c756c55d20ed3a52e099

Thanks